docs/AutoClone.md

```diff
@@ -82,28 +82,28 @@ cache.SetupSerializer(new FusionCacheSystemTextJsonSerializer());
 
 cache.Set("foo", new Person { Name = "John" });
 
-// RETURNS A CLONE OF CACHED INSTANCE (ORIGINAL REMAINS UNCHANGED)
+// RETURNS A CLONE OF THE CACHED INSTANCE
 var person1 = cache.GetOrDefault<Person>("foo", options => options.SetAutoClone(true));
 Console.WriteLine($"person1: {person1.Name}");
 Console.WriteLine();
 
-// RETURNS A CLONE OF CACHED INSTANCE (MODIFICATIONS AFFECT ONLY THE CLONE)
+// RETURNS A CLONE OF THE CACHED INSTANCE: CHANGES APPLIED ONLY THE CLONE, CACHED INSTANCE REMAINS UNCHANGED
 var person2 = cache.GetOrDefault<Person>("foo", options => options.SetAutoClone(true));
 person2.Name = "Jane";
 Console.WriteLine($"person1: {person1.Name}");
 Console.WriteLine($"person2: {person2.Name}");
 Console.WriteLine();
 
-// RETURNS DIRECT REFERENCE TO CACHED INSTANCE (MODIFICATIONS AFFECT THE CACHE)
+// RETURNS DIRECT REFERENCE TO THE CACHED INSTANCE: CHANGES APPLIED TO THE CACHED INSTANCE ITSELF
```

Comment:
Same as the comment above. Also, it's missing the verb "are" in "Changes _are_ applied to", which makes it verbose. 🤷